lua/nvim-paredit/utils/common.lua

```diff
@@ -39,4 +39,25 @@ function M.compare_positions(a, b)
   return -1
 end
 
+-- Removes all extra keys from t1 which is not in original
+-- and returns a new table
+--
+-- remove_extras({ "a", "b", "f", "d"}, {"a", "b", "c"}) => { "a", "b" }
+function M.remove_extras(tbl, original)
```

Comment:
Nit: I think `intersection` would be a better name here.

Reply:
will rename